html/payer/index.html

```diff
@@ -153,6 +153,23 @@ <h1>Jim C</h1>
           <div class="epitaph">The office is now less safe, without its volunteer police officer.</div>
         </div>
         <div class="clear"></div>
+      </div> 
+       
+      <div class="headstone">
+        <div class="grave">
+          <div class="grave-coffin"></div>
+        </div>
+        <div class="engraving">
+          <div class="header">
+            <div class="name">
+              <h1>C-LO</h1>
+            </div>
+            <div class="death_date">2019</div>
```

Comment:
We need to keep this list sorted from newest to oldest. Could you make sure this falls in the correct spot on that list?